openbb_terminal/DOCKER_README.md

```diff
@@ -0,0 +1,183 @@
+
+**DOCKER**
+
+If you don't have `Docker Compose` you can also use `Docker` directly to run the `OpenBBTerminal Docker Container`.
+
+Here is the commands to run:
+
+```bash
+docker pull ghcr.io/openbb-finance/openbbterminal/poetry:X.Y.Z
+
+docker run -v ~/.openbb_terminal/:/home/python/.openbb_terminal -v ~/OpenBBUserData:/home/python/OpenBBUserData -it --rm ghcr.io/openbb-finance/openbbterminal/poetry:X.Y.Z
+```
+
+Be sure to replace `X.Y.Z` with the version you want to pull and run.
+
+Note for windows:
+
+```text
+    Ignore this message if you are using Powershell or a more evolved interpreters. 
+    If you are using the builtin Windows interpreter.
+    Replace `~` by `%USERPROFILE%` in the command above.
+```
+
+## 3. Configuring your X-server to show plots
+
+In order to display plots in the docker container, we need to configure the XServer on the host machine.
+Without this configuration the interactive charts will not be displayed.
+
+### On Windows
+
+Download and install : [VcXsrv](https://sourceforge.net/projects/vcxsrv/)
+
+When running the program is important to check "Disable access control"
+
+Run `Docker Compose` like this:
+
+```bash
+curl -o docker-compose.yaml https://raw.githubusercontent.com/OpenBB-finance/OpenBBTerminal/main/docker/docker-compose.yaml
+
+docker compose run poetry
+```
+
+Or run `Docker` directly:
+
+```bash
+docker run -v ~/.openbb_terminal:/home/python/.openbb_terminal -v ~/OpenBBUserData:/home/python/OpenBBUserData -it --rm --env DISPLAY=host.docker.internal:0.0 ghcr.io/openbb-finance/openbbterminal/poetry:X.Y.Z
+```
+
+### X-Server on macOS
+
+Users familiar with Docker and X-Server can set the `DISPLAY` variable in the file [setenv](/docker/setenv) described above. If you use this approach remember to add `:0` at the end of your inet address. E.g. `DISPLAY=192.168.1.155:0`.
+
+For help setting up the X-Server continue reading:
+
+#### Setting up X Quartz/X11
+
+On macOS the X11 client of choice is [XQuartz](https://www.xquartz.org/). On Windows it's [Xming](http://www.straightrunning.com/XmingNotes/). XQuartz will be used as an example further on.
+
+0. Install X Quartz from <https://www.xquartz.org/>
+1. With X Quartz open: go to Preferences -> Security and make sure both options are enabled.
+   ![Screen Shot 2021-09-08 at 12 21 48 PM](https://user-images.githubusercontent.com/18151143/132548605-235d774b-9aa6-4a45-afcf-58fb775d376a.png)
+
+#### Adding the display for Docker
+
+From the command prompt or terminal, run the following to add your local configuration to the list of allowed access control:
+
+```bash
+IP=$(ifconfig | grep inet | grep -v -e "127.0.0.1" | awk '$1=="inet" {print $2}')
+xhost + $IP
+```
+
+Now we can run the docker container, adding the display to the environment:
+
+```bach
+docker run -it --rm --env-file=path/to/setenv --env DISPLAY=$IP:0 ghcr.io/openbb-finance/openbbterminal/poetry:X.Y.Z
```

Comment:
I think `--env-file=path/to/setenv` needs to be replaced with a volumes solution

Reply:
You are right.